modules/probe.sh

```diff
@@ -179,8 +180,9 @@ probe_initrd_image() {
     n=$(wc -l <<< "$initrds")
 
     if [ "$n" -gt "1" ];then
-      log_info "$n initrds found, selecting newest: $initrd_image_path"
-      log_info "Set MENDER_GRUB_INITRD_IMAGETYPE to override selected initrd"
+      log_warn "Found multiple initrds: $(awk '{print $2}' <<< $initrds)"
```

Comment:
`$initrds` -> `${initrds}`, for consistency.

Would also reword,

`Found multiple initrds:` -> `Found multiple initrd images:`